front/app/modules/commercial/content_builder/hooks/useContentBuilder.ts

```diff
@@ -0,0 +1,25 @@
+import { useState, useEffect } from 'react';
+import {
+  builderLayoutStream,
+  IBuilderLayout,
+} from '../services/ContentBuilder';
+
+const useBuilderLayout = (id: string, code: string) => {
```

Comment:
Would be nice to have an object here {id, code} instead of two separate arguments because that way they are named and you know what is expected for each one.